standard/standard/clause_7_normative_text.adoc

```diff
@@ -77,8 +73,7 @@ Classes, attributes, and relationships of the GeoPose domain are specified in a
 SDUs provide Standardization Targets that are independent of serialization/encoding format. This allows multiple equivalent serializations to be defined. Each SDU that may be expressed as a concrete data object is associated with a corresponding element (class or attribute) in the logical model.
 
 [[standardization_targets]]
```

Comment:
This anchor does not point to any reference. Do we really need it?

Reply:
Let’s get rid of it.